.github/workflows/acceptance-tests-on-emulator.yaml

```diff
@@ -18,16 +18,23 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        ruby: [2.5, 2.6, 2.7, 3.0]
+        ruby: [2.6, 2.7, 3.0]
```

Comment:
The presubmits test Ruby versions 2.6, 2.7 and 3.0, as those are the only ones recommended for use with ActiveRecord 6.x.